IBCMerge sporatically fails

PR: [#1180](https://github.com/dotnet/roslyn-internal/pull/1180) *Revert previous integration change now that open side has gone through.* by @CyrusNajmabadi
Failure: http://dotnet-ci.cloudapp.net/job/Private/job/dotnet_roslyn-internal/job/master/job/windows_vsi_p1_prtest/385/

**Issue 1: MSBuild Error**
error MSB3073: The command ""d:\j\workspace\windows_vsi_p---9267604d\Closed\Tools\ibcmerge\IBCMerge.exe" -q -f -partialNGEN -minify -mo "d:\j\workspace\windows_vsi_p---9267604d\Open\Binaries\Obj\EditorFeatures\Release\Microsoft.CodeAnalysis.EditorFeatures.dll" -incremental "C:\Users\dotnet-bot\.nuget\packages\RoslynDependencies.OptimizationData\1.3.0-beta5-60518-02\content\OptimizationData\Microsoft.CodeAnalysis.EditorFeatures.pgo"" exited with code -1. [d:\j\workspace\windows_vsi_p---9267604d\Open\src\EditorFeatures\Core\ep3fwyrh.tmp_proj]

--- a/build/Targets/VSL.Imports.targets
+++ b/build/Targets/VSL.Imports.targets
@@ -388,7 +388,7 @@
 
   <!-- The path to IBCMerge.exe must be specified at build time via $(IbcMergePath).  Typically this is passed as a build definition variable -->
   <Target Name="ApplyOptimizations"
-          Condition="'$(Configuration)' == 'Release' AND '$(NonShipping)' != 'true' AND '$(SkipApplyOptimizations)' != 'true' AND Exists('$(OptimizationDataFile)') AND Exists('$(IbcMergePath)')"
+          Condition="'$(OfficialBuild)' == 'true' AND '$(NonShipping)' != 'true' AND '$(SkipApplyOptimizations)' != 'true' AND Exists('$(OptimizationDataFile)') AND Exists('$(IbcMergePath)')"
           Inputs="@(IntermediateAssembly)"
           Outputs="@(IntermediateAssembly);$(PostCompileBinaryModificationSentinelFile)">
 

